framework/Driver.py

```diff
@@ -37,6 +37,14 @@
 import traceback
 import xml.etree.ElementTree as ET
 
+# for profiling method timing in python with kernprof
+import builtins
+try:
+  builtins.profile
+except (AttributeError,ImportError):
+  # profiler not preset, so pass through
+  builtins.profile = lambda f: f
```

Comment:
since this is a decorator, can we add this in RAVEN decorator module?

Reply:
I suppose we could; it would just mean adding that import to any file we want to use profiling on. I can see if I can mask it so that we have to import it through the decorators module.

Reply:
I added a section to the Python Optimization wiki page.